docs/reference/release-notes/6.8.asciidoc

```diff
@@ -1,6 +1,12 @@
 [[release-notes-6.8.9]]
 == {es} version 6.8.9
 
+[float]
+=== Known issues
+* Slow loggers are causing log4j loggers to leak over time. When new index is created, new log4j logger was associated with it.
+However when deleting an index it, log4j was keeping a reference internally to its loggers causing a memory leak.
+https://github.com/elastic/elasticsearch/issues/56171 Fixed in 6.8.10 and 7.7.1
```

Comment:
Slow loggers can cause Log4j loggers to leak over time. When a new index is created, a new Log4j logger is associated with it. However, when an index is deleted, Log4j keeps an internal reference to its loggers that results in a memory leak {pull}56171[#56171]

This issue is fixed in Elasticsearch 6.8.10 and 7.7.1.